bullet_train-themes/app/views/themes/base/attributes/_date.html.erb

```diff
@@ -1,11 +1,12 @@
 <% object ||= current_attributes_object %>
 <% strategy ||= current_attributes_strategy || :none %>
 <% url ||= nil %>
+<% local_assigns[:format] ||= :default %>
 <% default_message = local_assigns[:default_message] || t('global.formats.timestamp_unavailable') %>
 
 <% if object.send(attribute).present? %>
   <%= render 'shared/attributes/attribute', object: object, attribute: attribute, strategy: strategy, url: url do %>
-    <%= display_date(object.send(attribute), local_assigns[:date_format]) %>
+    <%= display_date(object.send(attribute), format: local_assigns[:format]) %>
```

Comment:
I wonder if we need the `nil` checks / `||= :default` if we can do this:

```suggestion
    <%= display_date(object.send(attribute), **local_assigns.slice(:format)) %>
```

Reply:
> I wonder if we need the `nil` checks / `||= :default` if we can do this:

Done!